### ISSUE TITLE
warning/Error while "##### Evaluate Test Set #####" in train_crfnet.py

### ISSUE THREAD

**Issue by RRK13** [TUMFTM/CameraRadarFusionNet#10]
Warning/Error in the "**##### Evaluate Test Set #####**" in _train_crfnet.py_

`Running network on 6 workers: N/A% (0 of 6830) |                                                                                       | Elapsed Time: 0:00:00 ETA:  --:--:--2020-06-12 04:27:17.081662: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node filtered_detections_2/map/while/strided_slice_4. Error: Pack node (filtered_detections_2/map/while/stack) axis attribute is out of bounds: 1 

2020-06-12 04:27:17.081708: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node filtered_detections_2/map/while/strided_slice_2. Error: Pack node (filtered_detections_2/map/while/stack) axis attribute is out of bounds: 1 

2020-06-12 04:27:17.081966: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes_2/strided_slice_2. Error: Pack node (boxes_2/stack) axis attribute is out of bounds: 2 

2020-06-12 04:27:17.081986: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes_2/strided_slice_3. Error: Pack node (boxes_2/stack) axis attribute is out of bounds: 2 

2020-06-12 04:27:17.082000: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes_2/strided_slice_4. Error: Pack node (boxes_2/stack) axis attribute is out of bounds: 2 

2020-06-12 04:27:17.082014: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes_2/strided_slice_5. Error: Pack node (boxes_2/stack) axis attribute is out of bounds: 2 

2020-06-12 04:27:17.238923: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node filtered_detections_2/map/while/strided_slice_4. Error: Pack node (filtered_detections_2/map/while/stack) axis attribute is out of bounds: 1 

2020-06-12 04:27:17.238966: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node filtered_detections_2/map/while/strided_slice_2. Error: Pack node (filtered_detections_2/map/while/stack) axis attribute is out of bounds: 1 

2020-06-12 04:27:17.239223: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes_2/strided_slice_2. Error: Pack node (boxes_2/stack) axis attribute is out of bounds: 2 

2020-06-12 04:27:17.239244: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes_2/strided_slice_3. Error: Pack node (boxes_2/stack) axis attribute is out of bounds: 2 

2020-06-12 04:27:17.239260: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes_2/strided_slice_4. Error: Pack node (boxes_2/stack) axis attribute is out of bounds: 2 

2020-06-12 04:27:17.239275: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes_2/strided_slice_5. Error: Pack node (boxes_2/stack) axis attribute is out of bounds: 2 `

Note: this warning/error does not stop the evaluation, it continues to evaluations (at Night and at Rain) and finishes the training successfully. But it would be good to know the issue related to the "##### Evaluate Test Set #####" in train_crfnet.py
